### PR TITLE
Add driver verifone in section host-to-host for vx805

### PR DESCRIPTION
**Pull request by DarkFrostnight**
Added driver Verifone in section host to host.

### CODE REVIEW

File: documentacion/host-to-host/README.md [hunk 1]
```diff
@@ -47,6 +47,17 @@ Para el desarrollo y pruebas del comercio Transbank entrega un set de pruebas qu
...
+<strong>Driver</strong>
+
+Estos equipos funcionan tanto con puerto serial RS232 y USB (Generalmente plug and play), para el cual puedes necesitar instalar un [driver de verifone](/files/verifone.zip). 
+
+Este driver es compatible con los siguientes sistemas operativos informados por Verifone:
+* Windows 10 de 32/64 bits
+* Windows 8/8.1 de 32/64 bits
+* Windows 7 de 32/64 bits
+
+Por normas PCI los comercios no deben utilizar un Sistema Operativo bajo obsolescencia, además es muy importante mantener su Sistema Operativo con los últimos parches instalado, esto principalmente por un tema de seguridad.
```
Comment (isabellaahc) [inline documentacion/host-to-host/README.md:59]:
```suggestion
Por normas PCI los comercios no deben utilizar un Sistema Operativo bajo obsolescencia, además es muy importante mantener su Sistema Operativo con los últimos parches instalados, esto principalmente por un tema de seguridad.
```

File: documentacion/host-to-host/README.md [hunk 2]
```diff
@@ -47,6 +47,17 @@ Para el desarrollo y pruebas del comercio Transbank entrega un set de pruebas qu
 Equipo Fijo USB o Serial Pinpad host
 <img src="/images/documentacion/host2host/vx805.png" alt="Verifone vx805">
 
+<strong>Driver</strong>
+
+Estos equipos funcionan tanto con puerto serial RS232 y USB (Generalmente plug and play), para el cual puedes necesitar instalar un [driver de verifone](/files/verifone.zip).
```
Comment (isabellaahc) [inline documentacion/host-to-host/README.md:52]:
```suggestion
Estos equipos funcionan tanto con puerto serial RS232 y USB (generalmente plug and play), para el cual puedes necesitar instalar un [driver de verifone](/files/verifone.zip). 
```

**Review by gonzunigad — APPROVED** [review]
Great work 👌

File: documentacion/host-to-host/README.md [hunk 3]
```diff
@@ -47,6 +47,17 @@ Para el desarrollo y pruebas del comercio Transbank entrega un set de pruebas qu
 Equipo Fijo USB o Serial Pinpad host
 <img src="/images/documentacion/host2host/vx805.png" alt="Verifone vx805">
 
+<strong>Driver</strong>
+
+Estos equipos funcionan tanto con puerto serial RS232 y USB (Generalmente plug and play), para el cual puedes necesitar instalar un [driver de verifone](/files/verifone.zip). 
+
+Este driver es compatible con los siguientes sistemas operativos informados por Verifone:
```
Comment (gonzunigad) [inline documentacion/host-to-host/README.md:54]:
```suggestion
Este driver es compatible con los siguientes sistemas operativos, informados por Verifone:
```

File: documentacion/host-to-host/README.md (hunk 1 above)
Comment (gonzunigad) [inline documentacion/host-to-host/README.md]:
```suggestion
Por normas PCI, los comercios no deben utilizar un Sistema Operativo bajo obsolescencia. Además es muy importante mantener su sistema operativo con los últimos parches instalado, esto principalmente por un tema de seguridad.
```

File: documentacion/host-to-host/README.md [hunk 4]
```diff
@@ -47,6 +47,17 @@ Para el desarrollo y pruebas del comercio Transbank entrega un set de pruebas qu
...
+<strong>Driver</strong>
+
+Estos equipos funcionan tanto con puerto serial RS232 y USB (generalmente plug and play), para el cual puedes necesitar instalar un [driver de verifone](/files/verifone.zip). 
+
+Este driver es compatible con los siguientes sistemas operativos, informados por Verifone:
+* Windows 10 de 32/64 bits
+* Windows 8/8.1 de 32/64 bits
+* Windows 7 de 32/64 bits
+
+Por normas PCI los comercios no deben utilizar un Sistema Operativo bajo obsolescencia, además es muy importante mantener su Sistema Operativo con los últimos parches instalados, esto principalmente por un tema de seguridad.
```
Comment (gonzunigad) [inline documentacion/host-to-host/README.md:59]:
```suggestion
Por normas PCI, los comercios no deben utilizar un sistema operativo bajo obsolescencia. Además es muy importante mantener su Sistema Operativo con los últimos parches instalados, esto principalmente por un tema de seguridad.
```

**Review by gonzunigad — APPROVED** [review]

**Pull request merged**